examples/graphql-cache-invalidation/graphql_cache_invalidation.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina/graphql;
+
+// Defines a `record` type to use as an object in the GraphQL service.
+type User readonly & record {|
+    int id;
+    string name;
+    int age;
+|};
+
+// Defines an in-memory table to store the profiles.
+table<User> key(id) users = table [
+    {id: 1, name: "Walter White", age: 50},
+    {id: 2, name: "Jesse Pinkman", age: 25}
+];
+
+service /graphql on new graphql:Listener(9090) {
+
+    @graphql:ResourceConfig {
+        cacheConfig: {}
+    }
+    resource function get name(int id) returns string|error {
+        if users.hasKey(id) {
+            return users.get(id).name;
+        }
+        return error("User not found");
+    }
```

Comment:
We can remove this resource method.

Reply:
Should I remove the other resource method? Because I have used the `name` field in the `invalidate()` API which is used in the `updateName` method.

Reply:
Maybe we can use `user` and `users` fields, and use `invalidate` when updating a `user`, and use `invalidateAll` when deleting a user?

Reply:
The `user` takes ID to filter and return a single user, `users` field can return an array of users.

Reply:
Okay, sure. That makes sense.